### PR TITLE
remove spelling mistakes in comments of C files (GCI task)

### PR DESCRIPTION
**Pull request by harpreetk1**

### CODE REVIEW

**Comment by cfsmp3** [comment]
We need a clean PR. Also small PRs are better than large PRs, as they can be validated and merged easily.

**Comment by harpreetk1** [comment]
What do you mean by a 'clean' PR?

**Comment by harpreetk1** [comment]
What do you mean by a 'clean' PR?

**Comment by harpreetk1** [comment]
What do you mean by a 'clean' PR?

**Comment by canihavesomecoffee** [comment]
Your pull request contains data we don't "want", and is unneeded.

File: src/gpacmp4/avilib.c [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /*
  *  avilib.c
  *
- *  Copyright (C) Thomas �streich - June 2001
- *  multiple audio track support Copyright (C) 2002 Thomas �streich
+ *  Copyright (C) Thomas �streich - June 2001
+ *  multiple audio track support Copyright (C) 2002 Thomas �streich
```
Comment (canihavesomecoffee) [inline src/gpacmp4/avilib.c]: Messed up the encoding of this file.

File: src/gpacmp4/av_parsers.c [hunk 2]
```diff
@@ -1743,8 +1743,8 @@ s32 AVC_ReadSeqInfo(char *sps_data, u32 sps_size, AVCState *avc, u32 subseq_sps,
 			u8 separate_colour_plane_flag = gf_bs_read_int(bs, 1);
 			/*
 			Depending on the value of separate_colour_plane_flag, the value of the variable ChromaArrayType is assigned as follows.
-			�	If separate_colour_plane_flag is equal to 0, ChromaArrayType is set equal to chroma_format_idc.
-			�	Otherwise (separate_colour_plane_flag is equal to 1), ChromaArrayType is set equal to 0.
+			�	If separate_colour_plane_flag is equal to 0, ChromaArrayType is set equal to chroma_format_idc.
+			�	Otherwise (separate_colour_plane_flag is equal to 1), ChromaArrayType is set equal to 0.
```
Comment (canihavesomecoffee) [inline src/gpacmp4/av_parsers.c]: Garbage data in here

File: src/cmake-build-debug/lib_ccx/cmake_install.cmake [hunk 3]
```diff
@@ -0,0 +1,96 @@
+# Install script for directory: /Users/Gurpreet/Desktop/CCextractor/src/lib_ccx
```
Comment (canihavesomecoffee) [inline src/cmake-build-debug/lib_ccx/cmake_install.cmake]: We don't need any file from the cmake-build-debug folder, that's not any source code.

File: src/gpacmp4/box_code_base.c [hunk 4]
```diff
@@ -1311,7 +1311,7 @@ GF_Err gnrm_Write(GF_Box *s, GF_BitStream *bs)
 	GF_Err e;
```
Comment (canihavesomecoffee) [inline src/gpacmp4/box_code_base.c]: Any spelling correction in gpacmp4 is quite useless, as that's an external library. It'll be gone next time we update gpacmp4

File: src/lib_ccx/ccx.pc [hunk 5]
```diff
@@ -0,0 +1,10 @@
+prefix=/usr/local
```
Comment (canihavesomecoffee) [inline src/lib_ccx/ccx.pc]: What's this file for?

File: src/gpacmp4/avilib.c (hunk 1 above)
Comment (harpreetk1) [inline src/gpacmp4/avilib.c]: What file encoding do you use?
Thanks.

**Pull request closed without merging**